packages/blade/src/components/Amount/types.ts

```diff
@@ -0,0 +1,5 @@
+import type { Theme } from '~components/BladeProvider';
+
+export type StyledAmountProps = {
+  theme: Theme;
```

Comment:
Why do we need to pass down theme? any component can get this directly by using `useTheme()`